to_sandbox.txt

```diff
@@ -39,4 +42,16 @@ lang/languages/bho_Kthi.textproto # https://github.com/google/fonts/pull/6573
 lang/languages/bku_Buhd.textproto # https://github.com/google/fonts/pull/6573
 lang/languages/doi_Takr.textproto # https://github.com/google/fonts/pull/6573
 lang/languages/hnn_Hano.textproto # https://github.com/google/fonts/pull/6573
+# Deleted: lang/languages/sa_Avst.textproto # https://github.com/google/fonts/pull/6573
+# Deleted: lang/languages/sa_Batk.textproto # https://github.com/google/fonts/pull/6573
+# Deleted: lang/languages/sa_Buhd.textproto # https://github.com/google/fonts/pull/6573
+# Deleted: lang/languages/sa_Dogr.textproto # https://github.com/google/fonts/pull/6573
+# Deleted: lang/languages/sa_Hano.textproto # https://github.com/google/fonts/pull/6573
+# Deleted: lang/languages/sa_Khoj.textproto # https://github.com/google/fonts/pull/6573
+# Deleted: lang/languages/sa_Kthi.textproto # https://github.com/google/fonts/pull/6573
+# Deleted: lang/languages/sa_Lepc.textproto # https://github.com/google/fonts/pull/6573
+# Deleted: lang/languages/sa_Mahj.textproto # https://github.com/google/fonts/pull/6573
+# Deleted: lang/languages/sa_Olck.textproto # https://github.com/google/fonts/pull/6573
+# Deleted: lang/languages/sa_Saur.textproto # https://github.com/google/fonts/pull/6573
+# Deleted: lang/languages/sa_Takr.textproto # https://github.com/google/fonts/pull/6573
```

Comment:
Shouldn't we also put these files in the to_delist.txt file as well?

Reply:
Thanks! I think so? Honestly the `to_delist` part I still don't understand fully.

Updating the PR now.

Reply:
Wait, the `to_delist.txt` file is currently exactly the above list.

Reply:
https://github.com/google/fonts/pull/6579/files